data/extra-install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+FRAMEWORKAPPS="
+	com.endlessm.dinosaurs.en
+	com.endlessm.encyclopedia.en
+"
+
+LOCKSCREENSDIR=/app/share/lockscreens
```

Comment:
It would be cool to respect the build prefix here.

Also I am not sure that this script is necessary to be honest, or the best way to solve this problem. The toolbox should already know that a framework app uses lockscreens called "framework", and I'd rather have the toolbox look for that asset directly rather than adding all these symlinks here.

@ptomato perhaps you can help suggesting how to do that.

Reply:
To respect the prefix in this script you can look in the environment variable `MESON_INSTALL_PREFIX`, it will be set when running in the script.

I think it would be difficult to special-case the lockscreen names. It's already determined that the locks are named per-app, since the key has to open each app's lock separately. You could name the assets according to the key IDs rather than the lock IDs, but then you would not be able to have a separate asset for the hackdex lock (because that is also opened by the framework key.) Basically I think there will have to be annoying architecture somewhere, and it might as well be here, so that we can keep the lockscreen asset names equal to the lock IDs.